stl/inc/xutility

```diff
@@ -1025,7 +1025,11 @@ struct _Distance_unknown {
 };
 
 template <class _Diff>
+#ifdef __cpp_lib_concepts // TRANSITION, GH-395
+_INLINE_VAR constexpr _Diff _Max_possible_v{static_cast<_Make_unsigned_like_t<_Diff>>(-1) >> 1};

```

Comment:
No change requested: `__cpp_lib_concepts` implies that `_INLINE_VAR` is `inline`, but it's reasonable to match the `#else` here.